app/assets/stylesheets/components/cards.scss

```diff
@@ -0,0 +1,50 @@
+@import '../vars/_variables.scss';
+
+.crayons-card {
+  border-radius: $br-default;
+  background: $smoke-00;
+
+  &__header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border-bottom: 1px solid $smoke-10;
+    padding: $su-2 $su-4;
+  }
+
+  &__title {
+    font-size: $fs-l;
+    color: $smoke-100;
+  }
+
+  &__body {
+    // padding: $su-4;
```

Comment:
Why did you decide to uncomment this line?

Reply:
because it's a work in progress and i needed that for testing locally and i actually haven't decided yet what's the best approach for setting up the padding.. there're two ways:

1. specifying default padding so it's always the same no matter what.
2. not specifying padding at all in CSS (so it's always 0) and then defining it in HTML with some additional class like `<div class="crayons-card__body padding-4">`